### DISCUSSION TITLE
Help required for setting custom logo.

### DISCUSSION THREAD

**Discussion by Tusharkumar0012** [filebrowser/filebrowser#1974] [Q&A]
Hi.
I installed filebrowser on docker . I am not clear about how to set custom branding. It is not clearly explained in the docs .

I am mounting
 - /home/username/filebrowser/config : /config 
in my docker-compose.yml & in config folder i have logo.svg file . But no change is reflected .

Could anyone please share what am I missing .

**Comment by ramiresviana** [comment]
The branding can only be set using the `config set` command. You can configure your container to run commands before starting the Filebrowser or use a existing database file with your configuration.

**Reply by Tusharkumar0012** [reply]
**### Noobs Guide to Custom Branding in Filebrowser**

**What enviornment am i using ?**
I am running docker & docker-compose on ubuntu server 22.04 . file browser is installed as a container using docker compose . i followed this https://docs.technotim.live/posts/meet-file-browser/ for setting filebrowser up.

in the docker-compose.yml file - add the following line  /home/serverandmin/filebrowser/config:/db/config 
Create a subdirectory with the name "img" in /home/serverandmin/filebrowser/config and place your logo in it after renaming it to "logo.svg" . Also if you want keep your css file with the name "custom.css" inside the config folder. Some css files are given in https://docs.theme-park.dev/themes/filebrowser/. ( ProTip - you can copy the logo file using filezilla sftp )


`
---
version: '3'
services:
  file-browser:
    image: filebrowser/filebrowser
    container_name: file-browser
    user: 1000:1000
    ports:
      - 8081:80
    volumes:
      - /home/serveradmin/:/srv                                                                         // serveradmin = username of your machine
      - /home/serveradmin/filebrowser/filebrowser.db:/database.db             // filebrowser = directory under which all files of thefilebrowser will be kept
      - /home/serverandmin/filebrowser/config:/db/config                             //     /db/config will automatically be created inside the container after running the docker-compose command.
    restart: unless-stopped
    security_opt:
      - no-new-privileges:true
`

After successfully running the docker container , we need to check if the config folder is mounted or not . For that we run -

docker container exec your_container_id ls -al /db/config 

it should give output as -
drwxrwxr-x    3    uid        gid          4096     Jun  7 02:16      .
drwxr-xr-x    3     root      root         4096     Jun  6 20:57      ..
-rwxrwxr--    1      uid       gid             73      Jun  7 02:16     custom.css
drwxrwxr-x    2     uid       gid          4096     Jun  7 01:41     img

it should contain custom.css & img . if thats the case then you are good to go .

Then login to your filebrowser ui and go in Settings>Global Settings> Scroll down to "Branding directory path" > and add "/db/config" in there and then press update . refresh the page to se effect . If you dont see any effect logout the ui and open it again in Icognito Tab as the browser cache the site and dont reflect updates

Cheers.
